clippy.toml

```diff
@@ -1,7 +1,13 @@
-# The default clippy value for this is 8 bytes, which is chosen to improve performance on 32-bit.
-# Given that kurbo is being designed for the future and already even mobile phones have 64-bit CPUs,
+# LINEBENDER LINT SET - .clippy.toml - v1
```

Comment:
Any chance you want to rename this file to `.clippy.toml` at the same time as the rest of this?

Reply:
Yes, good catch.